.github/copilot-instructions.md

```
@@ -0,0 +1,531 @@
+
+### Volt Class Based Component Example
+To get started, define an anonymous class that extends Livewire\Volt\Component. Within the class, you may utilize all of the features of Livewire using traditional Livewire syntax:
+
+
+<code-snippet name="Volt Class-based Volt Component Example" lang="php">
+use Livewire\Volt\Component;
+
+new class extends Component {
+    public $count = 0;
+
+    public function increment()
```

markdownlint-cli2 (0.17.2)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)